scripts/azure-pipelines/utils/AzureResourceUtils.psm1

```diff
@@ -464,7 +464,7 @@ function Remove-AzKeyVaultInResourceGroup {
     }
     else {
       Write-Host "Purging Key vault" $keyVault.VaultName "..."
-      Remove-AzKeyVault -VaultName $keyVault.VaultName -Location $keyVault.Location -ResourceGroupName $ResourceGroupName -InRemovedState -Force
```

Comment:
Are key vaults unique to a subscription? I'm surprised this works, but as long as we've checked it :)

Reply:
They are globally unique.